### ISSUE TITLE
Recreate Test set

### ISSUE THREAD

**Issue by marijnl** [KevinMusgrave/powerful-benchmarker#39]
I'm trying to recreate the split train/test exactly like in the experiment in raw python code.

These are my settings from `config_dataset.yaml`
```
split_manager:
  ClassDisjointSplitManager:
    test_size: 0.5
    test_start_idx: 0.5
    num_training_partitions: 4
    num_training_sets: 4
    hierarchy_level: 0
    data_and_label_getter_keys: null
```
Any (easy) way to do this?
I guess i would need to instantiate ClassDisjointSplitManager with the seed from the experiment but i dont know where i can find this info.

**Comment by KevinMusgrave** [comment]
Here's an example:

```python
from powerful_benchmarker.datasets import CUB200
from powerful_benchmarker.split_managers import ClassDisjointSplitManager
from torchvision import transforms


eval_transform = transforms.Compose([
                    transforms.Resize(256),
                    transforms.CenterCrop(227),
                    transforms.ToTensor(),
                    transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
                ])

dataset_root = "/scratch"

# first level of dictionary is transform type
# second level consists of separate dataset objects for each split
datasets = {"eval": {"train": CUB200(dataset_root, eval_transform),
                    "val": CUB200(dataset_root, eval_transform),
                    "test": CUB200(dataset_root, eval_transform)}
            }

split_manager = ClassDisjointSplitManager(num_training_partitions=4,
                                        num_training_sets=4,
                                        test_size=0.5,
                                        test_start_idx=0.5)


split_manager.create_split_schemes(datasets)


# get the test set
# first argument is the transform type
# second argument is split name
test_set = split_manager.get_dataset("eval", "test")

# for each split scheme, print the label set for each split
for name in split_manager.split_scheme_names:
    split_manager.set_curr_split_scheme(name)
    print("CURR SPLIT SCHEME =", split_manager.curr_split_scheme_name)
    print("TRAIN LABELS", split_manager.get_label_set("eval", "train"))
    print("VAL LABELS", split_manager.get_label_set("eval", "val"))
    print("TEST LABELS", split_manager.get_label_set("eval", "test"))
```

The split manager expects a nested dictionary, where the first level of keys correspond to transform types (i.e. the transforms you use during training vs evaluation), and the second level of keys correspond to the split names (train/val/test). The split managers all use torch.utils.data.Subset, which is a wrapper class that doesn't create a new dataset object. So to avoid weird things happening between different splits, I require that separate dataset objects are created from the very beginning.

**Comment by marijnl** [comment]
Great! I ran the code above and I can verify that it works.
I was able to reproduce my TEST performance.
